src/lib.rs

```diff
@@ -49,6 +49,11 @@ use std::path::Path;
 use std::time::{SystemTime, UNIX_EPOCH};
 use std::{fs, io};
 
+#[cfg(feature = "zip")]
+pub mod zip_chunk_provider;
+#[cfg(feature = "zip")]
+pub use zip_chunk_provider::*;
```

Comment:
As i understand this will automaticly import all items inside module `zip_chunk_provider` when `zip` feature are enabled. I think this is not correct.